test/spriter.test.js

```diff
@@ -98,6 +98,23 @@ describe('testing SVGSpriter', () => {
         }));
       });
 
+      it.each([
+        [null],
+        [undefined],
+        ['']
+      ])('should create vinyl file with file name if name is %p', name => {
```

Comment:
name here is always an array

Reply:
it.each([
        null,
        undefined,
        ''
      ])
This is how it should be done